examples/model_selection/plot_multi_metric_evaluation.py

```diff
@@ -0,0 +1,85 @@
+"""Demonstration of multi-metric evaluation on cross_val_score and GridSearchCV
+
+Multiple metric grid search (or random search) can be done by setting the
+``scoring`` parameter to a list of metric scorer names or a dict mapping the
+scorer names to the scorer callables.
+
+The scores of all the scorers are available in the ``cv_results_`` dict at keys
+ending in ``'_<scorer_name>'`` (``'mean_test_precision'``,
+``'rank_test_precision'``, etc...)
+
+The ``best_estimator_``, ``best_index_``, ``best_score_`` and ``best_params_``
+correspond to the scorer (key) that is set to the ``refit`` attribute.
```

Comment:
@amueller That is the current case, except `refit=False` and not the string `"False"`. Is that fine or do you want the behavior changed to `refit=False` by default for multiple metrics and `refit=True` by default for single metric?

Also cc: @jnothman @GaelVaroquaux